template/cordova/lib/PluginHandler.js

```diff
@@ -105,7 +105,7 @@ var handlers = {
             var type = obj.type;
 
             if(type === 'projectReference') {
-                project.removeProjectReference(path.join(plugin.dir, src), getTargetConditions(obj));
+                project.removeProjectReference(path.join('..', '..', 'plugins', plugin.id, src), getTargetConditions(obj));
```

Comment:
Regarding `targetDir` - this is [not supported](https://github.com/daserge/cordova-windows/blob/ee570036bc8faddc7d63e2830144c3d007d8e904/spec/unit/pluginHandler/windows.spec.js#L140) for projectReferences.
Regarding hardcoded `../..` - using `path.join(path.relative(project.projectFolder, plugin.dir), src)` instead is causing some issues with tests related to project and plugin different drives location, investigating.